examples/ota-requestor-app/linux/README.md

```diff
@@ -319,48 +319,108 @@ structs. Each entry in this list is a default OTA Provider per fabric. There can
 not be more than one entry containing the same fabric.
 
 To add more than one entry to the `DefaultOTAProviders` attribute, the OTA
-Requestor app must be commissioned on multiple fabrics. This can be done as
-specified below:
+Requestor app must be commissioned on multiple fabrics. At least one OTA
+Provider app should be commissioned to each corresponding fabric that the OTA
+Requestor app had been commissioned to.
```

Comment:
Please be consistent about "commissioned on" vs "commissioned to".  I think "commissioned on" is probably better....  Or "comissioned into"?

Reply:
Used "commissioned into" throughout